sample/src/samples/datepicker/advanced-options.js

```diff
@@ -1,3 +1,7 @@
+import {DatePickerDefaultParser} from '../../../../src/aurelia-materialize-bridge/datepicker/datepicker.default-parser';
```

Comment:
ah, did not come to mind. I didn't check if there was a aurelia-materialize-bridge alias in the config.js for the samples so I played it safe and did it directly ;) 
I can change tomorrow though :)